db/migrations/20240906090408_withdrawn_table.sql

```diff
@@ -0,0 +1,23 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE public.withdrawn (
```

Comment:
Существительное в множественном числе будет более консистентно с наименованиями остальных  колонок.

```suggestion
CREATE TABLE public.withdrawals (
```